.gitignore

```diff
@@ -12,3 +12,8 @@ bin/
 # Maven
 target/
 .flattened-pom.xml
+
+# Test data
+org.eclipse.transformer.cli/src/test/data/multi/input/
```

Comment:
This seems very wrong. The test case should never write into the source tree. I assume that is why you want to put ignore entries for things in the source tree here.

If the tests need to write somewhere, see https://stackoverflow.com/questions/12400243/what-is-the-correct-way-to-write-to-temp-file-during-unit-tests-with-maven for some ideas. A temp folder is probably the best choice.

Reply:
Yeah; that's a very improper practice.  (I've made this same suggestion many times.)  The initial code was following the pattern set by TestCommandLine.  I've added a new commit that changes the new tests and the already present TestCommandLine to place dynamic files under "target/test/data".  There were a few other data adjustments: Placing the test data more cleanly into test specific folders; Removing the now unnecessary "output" as a prefix from the output file name.

Reply:
You also need to remove these lines from .gitignore :-)

Reply:
Doh!  Fixed.